Sources/x10/xla_tensor/ops/cat.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
```

Comment:
Note: I believe the standard practice is to still use include guards instead of this. :-/

Reply:
I'm not sure we must strictly adhere to Google C++ style guide for this code and it's one less Copybara rule to worry about in the future. I'd rather keep these for now, `#pragma once` is widely supported.